Sources/PCRoom.swift

```diff
@@ -35,6 +36,12 @@ public final class PCRoom {
 
     public var createdAtDate: Date { return self.dateFormatter.date(from: self.createdAt)! }
     public var updatedAtDate: Date { return self.dateFormatter.date(from: self.updatedAt)! }
+    public var deletedAtDate: Date? {
```

Comment:
The type of this is different to the others. The force unwrapping I was doing with the others _shouldn't_ be problematic (given we know the format of timestamps that we're sending from the server), however, it does appear like there might be problems with it: https://github.com/pusher/chatkit-swift/issues/107

If we change one, we should change them all. Or not change any... I'd suggest following the format of the others and fixing any potential crash separately

Reply:
I am still force unwrapping the parse, the different signature is because `deletedAt` can be `nil`.

Reply:
Ah, sorry. Rushed the review and it shows 🤦‍♂️ Merge away